packages/eservice-template-process/test/vitestGlobalSetup.ts

```diff
@@ -0,0 +1,3 @@
+import { setupTestContainersVitestGlobal } from "pagopa-interop-commons-test/index.js";
```

Comment:
index.js not necessary
```suggestion
import { setupTestContainersVitestGlobal } from "pagopa-interop-commons-test";
```